### PR TITLE
chore: website landing page with animation duration state example

### PR DESCRIPTION
**Pull request by RikuVan**
experimenting with a landing page similar to Zustand with example

Screenshot 2022-04-10 at 21 03 04
Screenshot 2022-04-10 at 21 02 37

### CODE REVIEW

**Comment by RikuVan** [comment]
This was done is a bit of a rush and undoubtably needs polishing

File: website/pages/index.tsx [hunk 1]
```diff
@@ -1,29 +1,312 @@
...
+    state.baseDuration + "s"
+  );
+});
+
+const exampleCode = `
+  const state = proxy({
+    baseDuration: 4
+  });
+  const incDuration = () => {
+    state.baseDuration = ++state.baseDuration;
```
Comment (dai-shi) [inline website/pages/index.tsx:28]:
```suggestion
    ++state.baseDuration;
```
should be fine?

File: website/pages/index.tsx [hunk 2]
```diff
@@ -1,29 +1,312 @@
...
+    );
+  });
+  
+  <div>
+    <h3>
+      {snapshot.baseDuration}
+    </h3>
+    <button
+      disabled={snapshot.baseDuration <= 3}
+      onClick={() => decDuration()}>
```
Comment (dai-shi) [inline website/pages/index.tsx:47]:
```suggestion
      onClick={decDuration}>
```
should be fine?

**Review by dai-shi — COMMENTED** [review]
Thanks for working on this!
It's a bit unfortunate that the animation is reset when you click the inc/dec buttons.

File: website/pages/index.tsx (hunk 1 above)
Comment (RikuVan) [reply website/pages/index.tsx:28]: yeah of course

File: website/pages/index.tsx (hunk 2 above)
Comment (RikuVan) [reply website/pages/index.tsx]: yep, done

**Comment by RikuVan** [comment]
> Thanks for working on this! It's a bit unfortunate that the animation is reset when you click the inc/dec buttons.

Yeah I agree it is jarring. I suppose this could be done with a library that to smooth this out...

**Review by dai-shi — APPROVED** [review]
LGTM

**Pull request merged**